pytorch_lightning/accelerators/accelerator.py

```diff
@@ -383,12 +383,18 @@ def barrier(self, name: Optional[str] = None) -> None:
         self.training_type_plugin.barrier(name=name)
 
     def broadcast(self, obj: object, src: int = 0) -> object:
```

Comment:
shall we then rather call `source_rank`?

Reply:
Its called source in the lower level pytorch call that is commonly used outside of TPU/some training type plugin:

https://pytorch.org/docs/stable/distributed.html#torch.distributed.broadcast

So I think it might be alright to keep it as is!

Reply:
I vote for keep :)